### PR TITLE
Updated READMEs for the examples - Batch 1

### PR DESCRIPTION
**Pull request by andreasnaoum**
Updated READMEs for the examples:
Detect and Track Objects
Dicom MRI
Face Tracking
Gesture Detection
Human Pose Tracking
LiDAR
Live Camera Edge Detection
Live Depth Sensor

### Checklist
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)
* [x] I've included a screenshot or gif (if applicable)
* [x] I have tested the web demo (if applicable):
  * Using newly built examples: [app.rerun.io](https://app.rerun.io/pr/5620/index.html)
  * Using examples from latest `main` build: [app.rerun.io](https://app.rerun.io/pr/5620/index.html?manifest_url=https://app.rerun.io/version/main/examples_manifest.json)
  * Using full set of examples from `nightly` build: [app.rerun.io](https://app.rerun.io/pr/5620/index.html?manifest_url=https://app.rerun.io/version/nightly/examples_manifest.json)
* [x] The PR title and labels are set such as to maximize their usefulness for the next release's CHANGELOG
* [x] If applicable, add a new check to the [release checklist](https://github.com/rerun-io/rerun/blob/main/tests/python/release_checklist)!

- [PR Build Summary](https://build.rerun.io/pr/5620)
- [Docs preview](https://rerun.io/preview/53229bd36daf2158b782ef62b9d76bd90befbdee/docs) 
- [Examples preview](https://rerun.io/preview/53229bd36daf2158b782ef62b9d76bd90befbdee/examples) 
- [Recent benchmark results](https://build.rerun.io/graphs/crates.html)
- [Wasm size tracking](https://build.rerun.io/graphs/sizes.html)

### CODE REVIEW

**Comment by emilk** [comment]
Tip: configure your editor to trim trailing newlines. See https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md#tools for more :)

image

File: examples/python/detect_and_track_objects/README.md [hunk 1]
```diff
@@ -16,11 +17,158 @@ channel = "release"
...
+# Used Rerun Types
+[`Image`](https://www.rerun.io/docs/reference/types/archetypes/image), [`SegmentationImage`](https://www.rerun.io/docs/reference/types/archetypes/segmentation_image), [`AnnotationContext`](https://www.rerun.io/docs/reference/types/archetypes/annotation_context), [`Boxes2D`](https://www.rerun.io/docs/reference/types/archetypes/boxes2d), [`TextLog`](https://www.rerun.io/docs/reference/types/archetypes/text_log)
+
+
+# Logging and Visualizing with Rerun
+The visualizations in this example were created with the following Rerun code.
+
+
+## Timelines
+
```
Comment (emilk) [inline examples/python/detect_and_track_objects/README.md:31]:
```suggestion
```

File: examples/python/detect_and_track_objects/README.md [hunk 2]
```diff
@@ -16,11 +17,158 @@ channel = "release"
...
+```python
+def setup_logging() -> None:
+    logger = logging.getLogger()
+    rerun_handler = rr.LoggingHandler("logs")
+    rerun_handler.setLevel(-1)
+    logger.addHandler(rerun_handler)
+
+def main() -> None:
+    # .... existing code ....
```
Comment (emilk) [inline examples/python/detect_and_track_objects/README.md:145]:
```suggestion
    # … existing code …
```

File: examples/python/detect_and_track_objects/README.md [hunk 3]
```diff
@@ -16,11 +17,158 @@ channel = "release"
...
+same color.
+
+Note that it is also possible to log multiple annotation contexts should different colors and / or labels be desired.
+The annotation context is resolved by seeking up the entity hierarchy.
+
+## Text Log
+Rerun integrates with the [Python logging module](https://docs.python.org/3/library/logging.html). 
+Through the [`TextLog`](https://www.rerun.io/docs/reference/types/archetypes/text_log#textlogintegration) text at different importance level can be logged. After an initial setup that is described on the
+[`TextLog`](https://www.rerun.io/docs/reference/types/archetypes/text_log#textlogintegration), statements
+such as `logging.info("...")`, `logging.debug("...")`, etc. will show up in the Rerun viewer.
```
Comment (emilk) [inline examples/python/detect_and_track_objects/README.md:135]:
```suggestion
such as `logging.info("…")`, `logging.debug("…")`, etc. will show up in the Rerun viewer. 
```

**Review by nikolausWest — CHANGES_REQUESTED** [review]
These also all need background sections

**Review by Wumpf — APPROVED** [review]
background sections lgtm

File: examples/python/face_tracking/README.md [hunk 4]
```diff
@@ -15,44 +16,179 @@ thumbnail_dimensions = [480, 480]
...
+    for id_ in ids:
+        self._class_ids[id_] = i
+
+    # Append class description with class ID and keypoint connections
+    class_descriptions.append(
+        rr.ClassDescription(
+            info=rr.AnnotationInfo(id=i),
+            keypoint_connections=klass,
+        )
+    )
```
Comment (roym899) [inline examples/python/face_tracking/README.md:103]: `class`, also remove `klass` from dictionary again?

Comment (Wumpf) [reply examples/python/face_tracking/README.md:103]: confirmed, `class` doesn't work

Comment (roym899) [reply examples/python/face_tracking/README.md:103]: Oh, makes sense. I'd probably use `class_` instead, but it doesn't matter.

**Pull request merged**